pytest_doctestplus/plugin.py

```diff
@@ -912,13 +919,13 @@ def test_filter(test):
         return tests
 
 
-def write_modified_file(fname, new_fname, changes):
+def write_modified_file(fname, new_fname, changes, encoding=None):
```

Comment:
Should the default here be consistent with addoption default (utf-8)?

Reply:
Yes, it should be consistent, though I'm not sure if we should change the default to utf-8 or leave as is now?

Reply:
I decided to set `encoding=None` to avoid changing the behavior of the `write_modified_file` function since I am not familiar with your codebase. I wanted to ensure that the function behaves as it did before.

From a developer's perspective, I would prefer using  `"utf-8"` as the default encoding.

Reply:
any changes required?

i guess the default value of  `write_modified_file` is uncritical.

default value for cli option defaults to "utf-8" which will be passed on.